tests/unit/core/api/test_index.js

```diff
@@ -209,141 +190,6 @@ describe('api', () => {
     });
   });
 
-  describe('core/api.search()', () => {
-    function mockResponse(responseProps = {}) {
-      return createApiResponse({
-        jsonData: {
-          results: [
-            { slug: 'foo' },
-            { slug: 'food' },
-            { slug: 'football' },
-          ],
-        },
-        ...responseProps,
-      });
-    }
-
-    it('sets the lang, limit, page and query', () => {
-      // FIXME: This shouldn't fail if the args are in a different order.
-      mockWindow.expects('fetch')
-        .withArgs(`${apiHost}/api/v3/addons/search/?app=android&page=3&q=foo&lang=en-US`)
-        .once()
-        .returns(mockResponse());
-      return api.search({
-        api: { clientApp: 'android', lang: 'en-US' },
-        auth: true,
-        filters: { page: parsePage(3), query: 'foo' },
-      })
-        .then(() => mockWindow.verify());
-    });
-
-    it('normalizes the response', () => {
-      mockWindow.expects('fetch').once().returns(mockResponse());
-      return api.search({ api: {}, auth: true, filters: { query: 'foo' } })
-        .then((results) => {
-          expect(results.result.results).toEqual(['foo', 'food', 'football']);
-          expect(results.entities).toEqual({
-            addons: {
-              foo: { slug: 'foo' },
-              food: { slug: 'food' },
-              football: { slug: 'football' },
-            },
-          });
-        });
-    });
-
-    it('surfaces status and apiURL on Error instance', () => {
-      const url = `${apiHost}/api/v3/addons/search/?page=3&q=foo&lang=en-US`;
-      mockWindow.expects('fetch')
-        .withArgs(url)
-        .once()
-        .returns(mockResponse({ ok: false, status: 401 }));
-
-      return api.search({
-        api: { lang: 'en-US' },
-        auth: true,
-        filters: { page: parsePage(3), query: 'foo' },
-      })
-        .then(unexpectedSuccess, (err) => {
-          expect(err.response.status).toEqual(401);
-          expect(err.response.apiURL).toEqual(url);
-        });
-    });
-  });
-
-  describe('search api', () => {
```

Comment:
These search tests were duped, probably from an older refactor. I consolidated things and got rid of a few duplicated tests 👍